src/localConstants/dataTestIds.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum DataTestIdsEnum {
+  statusSubtitle = 'statusSubtitle',
```

Comment:
check across project for dataTestId and insert here

Reply:
In another PR/task